assets/js/notification-list.js

```diff
@@ -82,7 +88,7 @@ function checkAllButton(text, checked) {
     return button;
 }
 
-function updateRemoveChecked() {
+const updateRemoveChecked = () => {
     removeCheckedButton.disabled = !some.call(removeCheckboxes, isChecked);
     removeCheckedButtonTop.disabled = !some.call(removeCheckboxes, isChecked);
 }
```

Comment:
Missing semicolons (yeah, I know there’s no linter :P). Will probably fix and cherry-pick this conversion ahead of time.